scripts/configure

```diff
@@ -186,6 +186,10 @@ sed -i "s/tor.password=<password>/tor.password=$TOR_PASSWORD/g;" "$LND_CONF_FILE
 sed -i "s/TOR_PASSWORD=<password>/TOR_PASSWORD=$TOR_PASSWORD/g;" "$ENV_FILE"
 sed -i "s/TOR_HASHED_PASSWORD=<password>/TOR_HASHED_PASSWORD=$TOR_HASHED_PASSWORD/g;" "$ENV_FILE"
 
+# Store docker binary path
+DOCKER_BINARY=$(which docker)
```

Comment:
If we're trying to autodetect the path here maybe we should resolve any potential symlinks?

```suggestion
DOCKER_BINARY=$(readlink -f "$(which docker)")
```

Lets say `which docker` resolves to `~/bin/docker` which is a symlink to `/some/custom/install/dir/docker`. We'd end up mounting the symlink inside the container which would resolve to `/some/custom/install/dir/docker` inside the container and break.